app/src/main/res/layout-land/profile_list_fragment.xml

```diff
@@ -1,70 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
-
-  <data>
-
-    <variable
-      name="viewModel"
-      type="org.oppia.android.app.settings.profile.ProfileListViewModel" />
-  </data>
-
-  <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/white">
-
-    <com.google.android.material.appbar.AppBarLayout
-      android:id="@+id/profile_list_app_bar_layout"
-      android:layout_width="0dp"
-      android:layout_height="wrap_content"
```

Comment:
> that app bar in tablet mode is a layout configuration of the tablet emulater i am using . It has nothing to do with the code . if someone use different emulater for that then he wont get that app bar , nothing to worry with it PTAL @bkaur-bkj

@Rishavgupta12345 I see that you have merged the sw600dp (tablet) and mobile xml files without considering the fact that the tablet one doesn't have any `AppBarLayout` component. This means when we would be using `profile_list_fragment` for a tablet device, we will be seeing an extra AppBar in the screen (You can see that there is an unnecessary AppBar shown in the tablet screenshots)